@navikt/core/react/src/form/combobox/FilteredOptions/FilteredOptions.tsx

```diff
@@ -58,7 +58,9 @@ const FilteredOptions = () => {
         <ul
           ref={setFilteredOptionsRef}
           role="listbox"
-          className="navds-combobox__list-options"
+          className={cl("navds-combobox__list-options", {
+            "navds-combobox__list--max-selected": maxSelected?.isLimitReached,
```

Comment:
```suggestion
            "navds-combobox__list-options--max-selected": maxSelected?.isLimitReached,
```

Reply:
ah, jeg laget en ny class for å treffe parent med max-selected 😓 (så slapp man legge den på alle items)

Reply:
No biggie

Reply:
teknisk sett følger fortsatt BEM her, bare at element blir `list` og ikke `list-options` 🤔. Er nok litt BEM-rot her som kan ryddes i også tenker jeg.

Prøvde opprinnelig å løse dette med :has() og uten kodeendring, men fant ingen løsning der.

Reply:
Slik jeg forstår det, skal man bruke det opprinnelige navnet når man lager en modifier-klasse, her "navds-combobox__list-options". Det er dessuten allerede et element som heter "navds-combobox__list".

https://getbem.com/naming/
> Modifier is an extra class name which you add to a block/element DOM node. Add modifier classes only to blocks/elements they modify, and **keep the original class**.


Men :has() var en god idé, det burde jo funket... 🤔